docs/README.md

```diff
@@ -13,6 +13,20 @@ If you don't have `pnpm` installed, you will need to do so first.
 pnpm i
 ```
 
+## Optional Packages
+
+If you would like to include consent management functionality in your local development environment,
+you can install the `@wcp/wcp-consent` package. This package is used for handling user consent
+in certain scenarios. To install it, run:
+
+```shell
+pnpm i -w @wcp/wcp-consent
+```
+
+For details on how to authenticate and procure a PAT to use this package, please refer to the internal documentation.
```

Comment:
I would recommend linking to this documentation, and just noting that it is MS-internal.